ocaml/quicktest/quicktest_vm_import_export.ml

```diff
@@ -81,7 +70,7 @@ let import_export_test rpc session_id sr_info vm_template () =
                 if all.API.vBD_empty <> orig_vbd.API.vBD_empty
                 then Alcotest.fail (Printf.sprintf "Device %s varies in emptiness" all.API.vBD_userdevice);
                 match all.API.vBD_userdevice with
```

Comment:
I think I've said this before: a wildcard match would avoid runtime match exceptions.

Reply:
It's there on L86, isn't it?

Reply:
You are correct - so forget my comment